integration/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+Integration Testing
+===================
+
+All integration tests are in this module.
+
+Now integration testing supports three modes, the Local Standalone mode, the Cluster mode and the Remote mode.
+
+Integration Testing with Local Standalone Mode
+-------------------
+
+Integration testing with local standalone mode can be run with both maven and IDEs like IntelliJ.
+
+If you want run all tests include UT and IT, the maven command is:
+```
+mvn clean verify
+```
+
+Optionally, you can also only run IT, the maven command is:
+```
+mvn clean verify -pl integration
+```
+
+And if you want to run IT in the IDE like IntelliJ, pay attention to the `fork mode` in `run options`:
```

Comment:
If I want to run a test that is annotated with three modes.

What should I do to run LocalStandaloneTest, ClusterTest and RemoteTest separately?

Reply:
If you want to run any one of these three categories in the IDE like IntelliJ, like this:
![image](https://user-images.githubusercontent.com/67060864/143002197-619dae83-a58e-468e-afb6-20306b9872cc.png)

And if you want to run in maven cmd, use a profile that is already defined in integration/pom.xml.
Profile id with "LocalStandalone", "Remote" and "Cluster" can work well with these three categories separately.

In integration/pom.xml, the <test.includedGroups> part of each profile choose the category to run.
```
<profile>
    <id>LocalStandalone</id>   
    <properties>
         <test.includedGroups>org.apache.iotdb.itbase.category.LocalStandaloneTest</test.includedGroups>
         <test.excludedGroups/>
    </properties>
```